bot/cogs/core/paginator.py

```diff
@@ -11,13 +11,19 @@
 
 
 class PaginatorView(discord.ui.View):
-    def __init__(self, get_page, page_count, timeout):
+    def __init__(self, author, get_page, page_count, timeout):
```

Comment:
shouldn't make a big difference but we can also just pass the author_id instead of the whole object?

Reply:
https://github.com/Iapetus-11/Villager-Bot/pull/284/commits/11d9a0fb954b62eedb88e069bdfc04ea991c819f